src/Parsers/LatLongParserBase.php

```diff
@@ -224,7 +224,7 @@ protected function resolveDirection( string $coordinateSegment ): string {
 				$matches
 			);
 
-			if ( $matches[1] === $direction || $matches[3] === $direction ) {
+			if ( $matches[1] !== '' || $matches[3] !== '' ) {
```

Comment:
Looks like now it won't fail every time it should though. Better add some tests for that.

Reply:
#139 only touches the tests as far as I can see, so do I understand correctly that the code works as it should after all?

Reply:
Indeed.